extensions/amp-share-tracking/0.1/test/test-amp-share-tracking.js

```diff
@@ -89,7 +91,21 @@ describe('amp-share-tracking', () => {
   });
 
   it('should get outgoing fragment randomly if no vendor url ' +
-      'is provided', () => {
+      'is provided and win.crypto is availble', () => {
+    viewerForMock.onFirstCall().returns(Promise.resolve(''));
+    randomBytesMock.onFirstCall().returns(new Uint8Array([1, 2, 3, 4, 5, 6]));
+    return getAmpShareTracking().then(ampShareTracking => {
+      return shareTrackingForOrNull(ampShareTracking.win).then(fragments => {
+        // the base64url of byte array [1, 2, 3, 4, 5, 6]
+        expect(fragments.outgoingFragment).to.equal('AQIDBAUG');
+      });
+    });
+  });
+
+  it('should get outgoing fragment randomly if no vendor url ' +
+      'is provided and fallback to Math.random generation', () => {
+    sandbox.stub(Math, 'random').returns(0.1111111111111111);
```

Comment:
Pleas add some difference between the digits, so that the output isn't just `H`s.